src/components/propositionalLogic/propositionalLogic.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from "react";
+import { BrowserRouter, Route, Switch } from "react-router-dom";
+
+import ExercisesPropositionalLogic from ".//exercises";
+import ExamplesPropositionalLogic from "./examples";
+import FreeProverPropositionalLogic from "./freeProver";
+import NavigationPropositionalLogic from "./navigationPropositionalLogic";
+import TutorialPropositionalLogic from "./tutorial";
+
+// tslint:disable-next-line: no-empty-interface
+export interface Props {
+}
+
+// tslint:disable-next-line: no-empty-interface
+interface State {
+}
+
+class PropositionalLogic extends Component<Props, State> {
+	public render() {
+
+		return (
+			<BrowserRouter>
```

Comment:
You don't need the BrowserRouter here because all of this will inside the `App.tsx`'s BrowserRouter.

Reply:
i got it. thx i rebuild it inside the App.tsx and deleted the whole propositionalLogic.tsx